### ISSUE TITLE
dataclasses - operator== for I3RecoPulseSeriesMapMask (Trac #788)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#782]
Make the ==operator work as expected, especially in python, for all RecoPulse things.

Migrated from https://code.icecube.wisc.edu/ticket/788
```json
{
    "status": "closed", 
    "changetime": "2015-04-02T16:06:50", 
    "description": "Make the ==operator work as expected, especially in python, for all RecoPulse things.", 
    "reporter": "david.schultz", 
    "cc": "david.schultz", 
    "resolution": "fixed", 
    "_ts": "1427990810874730", 
    "component": "combo core", 
    "summary": "dataclasses - operator== for I3RecoPulseSeriesMapMask", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2014-10-20T16:15:05", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T21:21:35: 
* nega changed status from "new" to "assigned"
* nega changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20150402T16:06:50: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "assigned" to "closed"
* david.schultz commented:

> In changeset 130906:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="130906"
> add equality for Masks and Unions. fixes #782
> ```
